crates/nu-test-support/src/macros.rs

```diff
@@ -161,6 +161,7 @@ macro_rules! nu {
 
         let mut command = Command::new($crate::fs::executable_path());
         command
+            .env_clear()
```

Comment:
Can you replicate this in the other two `nu_with_std!`/`nu_with_plugins!` test macros? We sadly have a bunch of copy-pasted code here (I think this is still necessary due to the nature of macros invoked from macros)

Reply:
thanks for pointing this out, changed those as well